tutorial/lesson_07_multi_stage_pipelines.cpp

```diff
@@ -30,7 +31,7 @@ int main(int argc, char **argv) {
     // first horizontally, and then vertically.
     {
         // Take a color 8-bit input
-        Image<uint8_t> input = load<uint8_t>("images/rgb.png");
+        Image<uint8_t> input = load<Image<uint8_t>>("images/rgb.png");
```

Comment:
Do you think this syntax is a bit surprising? I can't decide if I like it or not. The ideal syntax for loading an image is probably:

Image<float> im = load_image("...");

I can think of some (slightly convoluted) ways that might make that work.